src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -0,0 +1,46 @@
+import isEmpty from 'ramda/src/isEmpty';
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import getAssetType from './getAssetType';
+import fetchPageData from '../../utils/fetchPageData';
+import { getMostReadEndpoint } from '#lib/utilities/getMostReadUrls';
+
+const pageTypeUrls = (assetType, service) => {
+  switch (assetType) {
+    case STORY_PAGE:
+      return [
+        {
+          name: 'mostRead',
+          path: getMostReadEndpoint({ service, variant }).replace('.json', ''),
```

Comment:
I believe this is the util function you mentioned in a previous comment. Shouldn't the `variant` be passed into this function as an argument?

Reply:
Previous comment: https://github.com/bbc/simorgh/pull/6338#discussion_r419523417

Reply:
it is now @mykeels `getMostReadEndpoint` expects an object, service is required variant has an optional default.